libs/community/langchain_community/vectorstores/__init__.py

```diff
@@ -109,6 +110,12 @@
 }
 
 
+def _import_couchbase() -> Any:
```

Comment:
where is this used?

Reply:
Sorry, it was from debugging. It is not used anywhere. I have removed it now. Could you please review again?